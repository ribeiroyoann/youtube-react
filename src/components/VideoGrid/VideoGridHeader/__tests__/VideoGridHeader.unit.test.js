import React from "react";
import { shallow } from "enzyme";
import { VideoGridHeader } from "../VideoGridHeader";

describe("VideoGridHeader", () => {
  test("renders VideoGridHeader no props", () => {
    const wrapper = shallow(<VideoGridHeader />);
    expect(wrapper).toMatchSnapshot();
  });
  test("renders VideoGridHeader empty title", () => {
    const wrapper = shallow(<VideoGridHeader title="" />);
    expect(wrapper).toMatchSnapshot();
  });
  test("renders VideoGridHeader with title", () => {
    const wrapper = shallow(<VideoGridHeader title="header" />);
    expect(wrapper).toMatchSnapshot();
  });
});
