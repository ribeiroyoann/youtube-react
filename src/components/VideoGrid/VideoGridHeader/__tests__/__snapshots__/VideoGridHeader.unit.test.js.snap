// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoGridHeader renders VideoGridHeader empty title 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;

exports[`VideoGridHeader renders VideoGridHeader no props 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;

exports[`VideoGridHeader renders VideoGridHeader with title 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  >
    header
  </span>
</div>
`;
